src/main/java/org/openpnp/model/Pad.java

```diff
@@ -3,6 +3,7 @@
 import java.awt.Shape;
 import java.awt.geom.Ellipse2D;
 import java.awt.geom.RoundRectangle2D;
+//import java.awt.geom.Path2D; //Used for Polygons as java.awt.Polygon only supports integer vertex
```

Comment:
Again, please don't comment out imports. Just remove them if you aren't using them.